src/pages/Home/Highlights.jsx

```diff
@@ -0,0 +1,24 @@
+import {Box, Typography} from "@mui/material";
+import s from './Home.module.css'
+import customColors, {defaultTheme} from "../../assets/styles";
+import React from "react";
+
+const HighlightsSection = ({highlights}) => (
```

Comment:
Do you expect the application to be mobile-friendly? If so I'd suggest allowing the highlights to stack vertically 
![image](https://github.com/Code-the-Dream-School/ee-prac-team3-front/assets/6400323/d76777f1-6511-4757-af2e-2029e622f6cd)

Reply:
The application is already mobile-friendly, at least I tried to do so. I didn't add the flex-wrap property on purpose. But if you think vertical will look better, then I can make changes.

Reply:
At first I tried to make them vertical, but then I decided to just make the font and images smaller when the screen size is reduced, since on some screen sizes it looks like this:
<img width="745" alt="Screenshot 2023-11-15 at 8 11 28 AM" src="https://github.com/Code-the-Dream-School/ee-prac-team3-front/assets/111474106/89981189-340b-4d1e-8965-69a884686963">

Reply:
Ah yes great call on making the icons smaller! I'm testing down to 320px wide, which is the minimum width a site needs to be AA accessible. So that's why I suggested vertically stacking.